DZ-3Kosulin/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DZ_3Kosulin
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int a = int.Parse(Console.ReadLine());
+            Console.Write(a / 100 +""+ a % 10 + ""+ a / 10 % 10);
```

Comment:
В задании сказано

> Найти полученное число, полученное..

Вы не находите число. Вы формируете строку, которой записывается это число. То есть не только разобрать число на сотни, десятки и единицы, но затем собрать новое число в соответствии с заданием. Например, так
```suggestion
            int number = (a / 100) * 100 + (a % 10) * 10 + a /10 % 10;
            Console.Write(number);
```